drupal/code/modules/features/pori_configuration_overrides_feature/pori_configuration_overrides_feature.features.inc

```diff
@@ -1175,7 +1175,8 @@ function pori_configuration_overrides_feature_default_scald_context_config_alter
 /**
  * Implements hook_strongarm_alter().
  */
-function pori_configuration_overrides_feature_strongarm_alter(&$data) {
+function pori_configuration_overrides_feature_strongarm_alter(&$data)
+{
```

Comment:
Should be formatted as it was (the curly bracket in the end of the row) per coding standards.